### PR TITLE
Add Intron Analysis Bellcurve component

### PR DESCRIPTION
**Pull request by mak2508**
Resolves issue #4.

### CODE REVIEW

File: src/client/BellCurveChart.js [hunk 1]
```diff
@@ -125,7 +125,7 @@ class BellCurveChart extends React.Component {
               layout: 'vertical',
               itemMarginTop: 5,
               itemMarginBottom: 5
-            },
+            },
```
Comment (fabubaker) [inline src/client/BellCurveChart.js]: Can you fix linter issues? Here's a trailing whitespace.

File: src/client/IntronAnalysisBellCurveChart.js [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+              layout: 'vertical',
+              itemMarginTop: 5,
+              itemMarginBottom: 5
+            },
+            xAxis: [{
+              title: {
+                text: ''
+              }
+            }],
+            yAxos: [{
```
Comment (fabubaker) [inline src/client/IntronAnalysisBellCurveChart.js]: Typo? Shouldn't it be `yAxis`?

File: src/client/IntronAnalysisBellCurveChart.js [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+              zoomType: 'x'
+            },
+            legend: {
+              align: 'right',
+              verticalAlign: 'top',
+              layout: 'vertical',
+              itemMarginTop: 5,
+              itemMarginBottom: 5
+            },
+            xAxis: [{
```
Comment (fabubaker) [inline src/client/IntronAnalysisBellCurveChart.js]: The X-Axis title should be `Log2 Psi` and the Y-Axis should be `Frequency`.

File: src/client/IntronAnalysisBellCurveChart.js [hunk 4]
```diff
@@ -124,6 +124,8 @@ class IntronAnalysisBellCurveChart extends React.Component {
     var series = []
     axios.all(requests)
       .then(axios.spread((...responses) => {
+        // key parameter expected by antd
+        let medianPsi = { 'key': 1 }
```
[the remark this answers is not in the document]
Comment (mak2508) [reply src/client/IntronAnalysisBellCurveChart.js]: I moved this to the parent but didn't see a need for the lodash function? The function is for merging two objects based on identical keys no? Let me know if there is an issue with the way I did things

File: src/server/adapter.js [hunk 5]
```diff
@@ -190,14 +190,15 @@ module.exports = {
   },
 
   intronAnalysisVertical: function (gene) {
+    console.log(gene)
```
Comment (mak2508) [inline src/server/adapter.js]: sorry I'll remove this

File: src/client/IntronAnalysisBellCurveChart.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (fabubaker) [reply src/client/IntronAnalysisBellCurveChart.js]: Nevermind, I thought you might need to merge. This looks good.

**Pull request merged**